openpype/hosts/houdini/plugins/publish/validate_primitive_hierarchy_paths.py

```diff
@@ -103,4 +113,62 @@ def get_invalid(cls, instance):
                 "Prims have no value for attribute `%s` "
                 "(%s of %s prims)" % (path_attr, len(invalid_prims), num_prims)
             )
-            return [output_node.path()]
+            return [output_node]
+
+    @classmethod
+    def repair(cls, instance):
+        """Add a default path attribute Action.
+
+        it's a helper action more than a repair action,
+        which used to add a default single value.
+        """
+
+        rop_node = hou.node(instance.data["instance_node"])
+        output_node = rop_node.parm('sop_path').evalAsNode()
+
+        if not output_node:
+            cls.log.debug(
+                "Action isn't performed, empty SOP Path on %s"
+                % rop_node
+            )
```

Comment:
Technically it might not be empty - it could also point to a non-existent path and then return None. Right?
```suggestion
            cls.log.debug(
                "Action isn't performed, invalid SOP Path on %s"
                % rop_node
            )
```

Reply:
Technically, you are right 👌